src/utils/util.js

```diff
@@ -0,0 +1,102 @@
+const crypto = require('crypto');
+
+module.exports = class Util {
+  /**
+   * Get the lowest smartrate of this shipment.
+   * @param {Object} smartrates
+   * @param {number} delivery_days
+   * @param {string} delivery_accuracy
```

Comment:
```suggestion
   * @param {number} deliveryDays
   * @param {string} deliveryAccuracy
```
These should be in camelCase